pkg/api/install/install_test.go

```diff
@@ -57,7 +57,7 @@ func TestCodec(t *testing.T) {
 	if err := json.Unmarshal(data, &other); err != nil {
 		t.Fatalf("unexpected error: %v", err)
 	}
-	if other.APIVersion != latest.GroupOrDie("").Version || other.Kind != "Pod" {
+	if other.APIVersion != latest.GroupOrDie("").GroupVersion.Version || other.Kind != "Pod" {
```

Comment:
IIUC, you mean because we know there is only one version for the internal group, we can use api.SchemeGroupVersion.Version here. But in this case, using the GroupOrDie("") has more readability, because people not necessarily know `api` is the internal version.

In general, there are two disadvantages of using SchemeGroupVersion directly:
1. Each group/version has a SchemeGroupVersion. To use it, one needs to call e.g., v1.SchemeGroupVersion.Version, which hardcodes the version in the package.
2. calling latest.GroupOrDie("group") also checks if the user has enabled the "group", I think this is necessary in most cases.

Perhaps there are cases where directly calling SchemeGroupVersion makes sense, pleases cc me when you have an example.